app/move/app/view/middleware/locals.feedback-url.js

```diff
@@ -0,0 +1,11 @@
+const { MOVE_DESIGN_FEEDBACK_URL } = require('../../../../../config')
+
+function setFeedbackUrl(req, res, next) {
+  if (MOVE_DESIGN_FEEDBACK_URL) {
```

Comment:
I think we'll need to update this to also check for the cookie being set to `1` like in [checkPreviewChoice](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/7ca8fd3624f3afa5ad86ccb708069f16eb7b6baa/app/move/middleware.js#L18)